src/components/consul/selectors/ImageSelector.js

```diff
@@ -64,6 +81,68 @@ export const ImageSelector = ({ control, field, item, imageId }) => {
     setInfoAndErrorText({});
   };
 
+  if (isVolunteer) {
+    return (
+      <>
+        <Input {...item} control={control} hidden name={name} value={JSON.stringify(value)} />
+        <RegularText smallest placeholder>
+          {infoText}
+        </RegularText>
+
+        <Button
+          title={buttonTitle}
+          invert
+          onPress={async () => {
+            const { uri, type } = await selectImage();
+            const { size } = await FileSystem.getInfoAsync(uri);
+
+            /* the server does not support files more than 10MB in size. */
+            const errorText = size > 10485760 && texts.volunteer.imageGreater10MBError;
+
+            /* used to specify the mimeType when uploading to the server */
+            const imageType = IMAGE_TYPE_REGEX.exec(uri)[1];
+
+            /* variable to find the name of the image */
+            const uriSplitForImageName = uri.split('/');
+            const imageName = uriSplitForImageName[uriSplitForImageName.length - 1];
+
+            setImagesAttributes([...imagesAttributes, { uri, mimeType: `${type}/${imageType}` }]);
+
+            setInfoAndErrorText([
+              ...infoAndErrorText,
+              {
+                errorText,
+                infoText: `${imageName}`
+              }
+            ]);
+          }}
+        />
+
+        {value
+          ? JSON.parse(value).map((item, index) => (
+              <View key={index}>
+                {!!infoAndErrorText[index]?.errorText && (
+                  <RegularText smallest error>
+                    {infoAndErrorText[index].errorText}
+                  </RegularText>
+                )}
```

Comment:
![image](https://user-images.githubusercontent.com/1942953/183450949-70fc03a8-90be-4896-93de-abefaf98c1a0.png)


please put this below the image as we always have validations below fields, example title:

![image](https://user-images.githubusercontent.com/1942953/183450926-2bb923b2-4e18-4993-9175-a62122f91d9c.png)

Reply:
![Simulator Screen Shot - iPhone 13 - 2022-08-09 at 08 39 31](https://user-images.githubusercontent.com/11755668/183581490-ad84caaa-bf3c-4576-ae5f-02f4eb1b94f5.png)

https://github.com/ikuseiGmbH/smart-village-app-app/pull/606/commits/8f4269d2017e14fcc104932249cffd5f1c467b34
https://github.com/ikuseiGmbH/smart-village-app-app/pull/606/commits/ba048ed71a501a766f49e70c8bc6dbc33e9c673f